docs/platforms/javascript/guides/nuxt/features/vue-router.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Vue Router
+description: "Learn about Sentry's Vue Routing integration."
+---
+
+Nuxt uses the vue-router by default, and our Nuxt SDK instruments your routing automaitcally, to parameterize transaction according to your routes.
+
+Our routing instrumentation supports `vue-router` v2, v3, and v4.
```

Comment:
```suggestion
Nuxt uses the `vue-router` under the hood, and our Nuxt SDK instruments your routing automatically, to parameterize transaction according to your routes.
```

In my opinion, we don't need to include the version support, as users don't install the vue-router themselves.